src/js/steps/letters/generateOverlay.js

```diff
@@ -168,16 +168,16 @@ module.exports = function (options, $events) {
        */
       function revertCharsToOriginal(callback) {
         data.combinedLetters
-          .map((letter, index) => ({ ...letter, index }))
+          .map((letter, index) => $.merge(letter, { index }))
```

Comment:
well, isn't $.merge for smushing together 2 arrays and removing duplicate values?

`{...letter, index}` creates a new object with a deep copy of `letter` and another property `index` with the literal key `index` and the object `index` as the actual value.